### ISSUE TITLE
please add link to .txt file for test embryos (under Models section... probably beneath where the .zip download links are)

### ISSUE THREAD

**Issue by haysconun** [flatironinstitute/blastoSPIM_web#24]
Could you add a link to a .txt file containing the following info?

The combined moderate SNR test set contains:
F24_1
F24_2
F24_6
F25_2
F25_8
F27_10
F27_7
F27_9
F29_3
F29_4
F30_4
F30_8
F30_9
M6_21
M6_12
M7_4
M7_0
F42_63
F41_56
F34_73
F33_67
F26_8
F24_10
F8_72
F44_87
F44_89
F39_117
F40_136
F49_148
F55_185
Blast_19
Blast_25
Blast_39
Blast_75
Blast_4

The combined low SNR test set contains:
F11_70
F11_71
T1_3
T1_4
H1_3
H1_5
H1_6
H2_11
H2_12
H3_0
H3_2
H3_7
H3_9
H5_4
H5_7
H5_9
H6_16
H7_2
H7_4
H7_7
H7_8
H8_11
H8_16
H8_18
H9_0
H9_1
H9_10
H9_8
M14_16
M3_2
M3_3
M3_8
M4_11
M4_12
M4_13
M4_17
M14_18
M14_19
M14_20
H2_16
H8_21
H4_11
H4_12
F16_68
F16_69
F16_70
F16_72
H4_13
M10_12
M10_15
M10_18
M7_7
M7_8
M8_14
M8_15
M8_16
F9_71
F9_74
F2_13
F2_12
Blast_35
Blast_74
Blast_77

The validation set for training on BlastoSPIM 1.0 (because we train and validate by patching images, for some of these images part of the image is in training while part of the image is in validation):
T2_15
T2_6
F26_2
F30_0
M6_16
M6_18
F30_5
F30_1
T2_9
F28_0
F30_7
F28_7
M6_15
M6_11
F26_5
F27_1
F32_52
F33_70
F19_67
M7_2
F42_65
F41_53
F33_61
F19_66
F38_105
F38_109
F46_107
F44_94
F40_138
F49_146
F55_186

The validation set for training on BlastoSPIM 2.0 (because we train and validate by patching images, for some of these images part of the image is in training while part of the image is in validation):
Blast_22
Blast_12
Blast_40
Blast_28

Both for training on BlastoSPIM 1.0 and for training on BlastoSPIM 2.0, the images not listed in the test sets above are used for training or validation.

**Comment by AaronWatters** [comment]
I can do this easily, but I'm thinking it might make more sense to include these
comments in a README in the zip files directly (?).

Let's chat about this before I make the change.
